tasks.py

```diff
@@ -115,8 +115,9 @@ def tutorials(c):
 @task
 def lint(c):
     check_dependencies(c)
-    c.run('flake8 copulas')
-    c.run('flake8 tests --ignore=D,SFS2')
+    c.run('flake8 copulas --ignore-names=X,U,V,W,A,B,T1,X_left_right,X_right_left,' \
```

Comment:
@csala Not sure what we want to do on this one. Lint complains that they start with capital letters, but we since they are matrices, they should be capitalized.

Reply:
As discussed on call, the ignores are fine, but we should move them to `setup.cfg`